### PR TITLE
Ins0970 atkana

### PR DESCRIPTION
**Pull request by MarcinKrawczuk**
I have still very little experience with institutions so I would really appreciate any suggestions, if necessary. Thanks!

### CODE REVIEW

File: new/INS0970Atkana.xml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            <t:change xmlns:t="http://www.tei-c.org/ns/1.0" who="MKr" when="2021-12-15">Created entity</t:change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPlace>
+                <place type="church religiousSite" subtype="institution">
+                    <placeName xml:lang="gez" xml:id="n1">አትከና፡ ጊዮርጊስ፡</placeName>
+                    <placeName xml:lang="gez" type="normalized" corresp="#n1">ʾAtkanā Giyorgis</placeName>
+                    <region notBefore="1991" type="RegionalState" ref="LOC6568Amhara"/>
```
Comment (eu-genia) [inline new/INS0970Atkana.xml:50]: add country` <country ref="LOC3010Ethiop"/>`

File: new/INS0970Atkana.xml [hunk 2]
```diff
@@ -49,8 +49,12 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                     <placeName xml:lang="gez" type="normalized" corresp="#n1">ʾAtkanā Giyorgis</placeName>
                     <region notBefore="1991" type="RegionalState" ref="LOC6568Amhara"/>
                      <settlement when="2021-12-15" type="warada" ref="LOC3056Farta"/>
+                    <note>
+                    <date type="foundation" when="1596">1596</date>
+                    <desc type="foundation">Founded by the rās <persName ref="PRS11898Waldakrestos">Walda Krǝstos</persName> in 1596.</desc>
+                    </note>
```
Comment (eu-genia) [inline new/INS0970Atkana.xml:55]: in note one could also add (in the note tag)
```
<ab type="tabot">
                  Main tābot dedicated to <persName ref="PRS4585Georgeo" type="tabot">George</persName>.
                  </ab>
```

File: new/INS0970Atkana.xml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPlace>
+                <place type="church religiousSite" subtype="institution">
+                    <placeName xml:lang="gez" xml:id="n1">አትከና፡ ጊዮርጊስ፡</placeName>
+                    <placeName xml:lang="gez" type="normalized" corresp="#n1">ʾAtkanā Giyorgis</placeName>
+                    <region notBefore="1991" type="RegionalState" ref="LOC6568Amhara"/>
+                     <settlement when="2021-12-15" type="warada" ref="LOC3056Farta"/>
```
Comment (eu-genia) [inline new/INS0970Atkana.xml:51]: one could add that previously it belonged to Bademder

File: new/INS0970Atkana.xml [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+            </t:langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <t:change xmlns:t="http://www.tei-c.org/ns/1.0" who="MKr" when="2021-12-15">Created entity</t:change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPlace>
+                <place type="church religiousSite" subtype="institution">
```
Comment (eu-genia) [inline new/INS0970Atkana.xml:47]: is it known which church type it is? could one add dabr to the type?

File: new/INS0970Atkana.xml [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+                    <placeName xml:lang="gez" xml:id="n1">አትከና፡ ጊዮርጊስ፡</placeName>
```
Comment (eu-genia) [inline new/INS0970Atkana.xml:48]: Jesuits wrote Atqhana for ʾAtkanā (e.g. here https://archive.org/details/rerumaethiopica00unkngoog)
(this made the authors of this book https://brill.com/view/book/9789004324695/B9789004324695_004.xml believe it as ʾAtḵanā - which it is not of course, but still the Portuguese spelling could be mentioned somewhere)

File: new/INS0970Atkana.xml (hunk 1 above)
Comment (eu-genia) [reply new/INS0970Atkana.xml:50]: coordinates 
```
<location>
                  <geo>11.966675 38.18329</geo>
               </location>
```
(taken from The Archaeology of the Jesuit Missions in Ethiopia; I assume the mission was established close to the AG church)

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/INS0970Atkana.xml (hunk 2 above)
Comment (MarcinKrawczuk) [reply new/INS0970Atkana.xml:55]: Done

File: new/INS0970Atkana.xml (hunk 4 above)
Comment (MarcinKrawczuk) [reply new/INS0970Atkana.xml:47]: I haven't found this info

File: new/INS0970Atkana.xml (hunk 5 above)
Comment (MarcinKrawczuk) [reply new/INS0970Atkana.xml:48]: OK, added the variant and bbg reference

File: new/INS0970Atkana.xml (hunk 3 above)
Comment (MarcinKrawczuk) [reply new/INS0970Atkana.xml:51]: What should the type be?

Comment (eu-genia) [reply new/INS0970Atkana.xml]: region prior to 1974
possibly also warada prior to 1991, have to check tmrw

**Review by thea-m — APPROVED** [review]

**Pull request merged**